docs/manual/docs/cmd/yammer/message/message-get.md

```diff
@@ -0,0 +1,38 @@
+# Yammer message get
+
+Returns a Yammer message
+
+## Usage
+
+```sh
+yammer message get [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`--id`|The id of the Yammer message
```

Comment:
Please add `<id>` to highlight that it is a mandatory none switch option. Otherwise, it looks like the switch option. Like 
```
`--id <id>`|The id of the Yammer message
```